### PR TITLE
Upgrade template, resigning Unverified commit

### PR DESCRIPTION
**Pull request by pacrob**
## What was wrong?

The project template was updated. This pulls in those changes. This is a re-do of PR #49 , re-signing what is now commit fe19f5a with a valid gpg key

[//]: # (See: https://eth-hash.readthedocs.io/en/latest/contributing.html#pull-requests)
- [x] Add entry to the [release notes](https://github.com/ethereum/eth-hash/blob/master/newsfragments/README.md)

#### Cute Animal Picture

image

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -46,22 +46,52 @@ extras=
 allowlist_externals=make
 
 [common-lint]
-# mypy needs cryptodome installed to infer types
```
Comment (kclowes) [inline tox.ini:49]: nit: this is nice background info IMHO

**Review by kclowes — APPROVED** [review]
Looks good to me!

File: setup.py [hunk 2]
```diff
@@ -63,6 +65,9 @@
     author_email="snakecharmers@ethereum.org",
     url="https://github.com/ethereum/eth-hash",
     include_package_data=True,
+    install_requires=[
```
Comment (kclowes) [inline setup.py:68]: I just noticed this - Looks like we dropped the eth-utils dependency here: https://github.com/ethereum/eth-hash/pull/33

Comment (pacrob) [reply setup.py]: noted, removed.

**Pull request merged**